<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1783</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="Table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>951</width>
      <height>491</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>130</number>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Expression</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>a</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>b</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>c</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>d</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>e</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Result</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="1" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="2" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="3" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="4" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="5" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="6" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="7" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="8" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="9" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="10" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="11" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="11" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="12" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="12" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="13" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="13" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="14" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="14" column="6">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ShowTable">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowExpressions">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>70</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Expressions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowVariables">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>130</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Variables</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CalculateResults">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>190</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Results</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="Table2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>1061</width>
      <height>61</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Your expression</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>a</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>b</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>c</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>d</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>e</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Result</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>a + b + c</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="6">
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="CalculateYourResult">
    <property name="geometry">
     <rect>
      <x>1084</x>
      <y>523</y>
      <width>521</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Your Expression</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1783</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
